src/Router/index.jsx

```diff
@@ -30,9 +30,17 @@ export const router = createBrowserRouter([
         )
       },
       {
-        path: 'product/:id', 
-        element: <ProductDetail />, 
-      }
+        path: 'product/:id',
+        element: <ProductDetail />,
+      },
+      {
+        path: 'ShoppingCart',
```

Comment:
According to what Lucia recommended, the paths should be in lowercase. In this case, I understand that it should be `shopping-cart`. 
It would be good to put it correctly in this one that we added, since the others are being fixed in the PR that I am working on.